docs/src/code-examples/sending-texts/python.mdx

```diff
@@ -65,25 +65,23 @@ A unique identifier you create. This reference identifies a single unique notifi
 If successful, the shape of the response you recieve will resemble the
 following.
 
-
 ```json
 {
   "content": {
-    "body": "Hey there. Lisa\n\nPlease pay us $15.50 at your earliest convenience.\n", 
```

Comment:
Out of interest did Prettier remove the newlines inside here or you? It's fine without them (neater), but probably not great if prettier is removing them...

Reply:
I removed these